### DISCUSSION TITLE
[Feature] Scale airport circles with map zoom

### DISCUSSION THREAD

**Discussion by palitu** [johanohly/AirTrail#27] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

i have a cluster of airports within the North West of Australia, and with the current circle size, they all overlap in a not-so-pretty way when zoomed out.

Scale the circles as you zoom out.

I will try and get a screen shot and attach.

**Comment by johanohly** [comment]
The main issue with this is most people log their international flights with this app, which results in a map looking similar to this
![image](https://github.com/user-attachments/assets/ddc644b9-82f0-4233-a59f-e531b0020aa2)
Granted, I don't fly a lot, but I can't imagine the circles being smaller fixing more issues than it brings up.
